doc/citing.md

```diff
@@ -28,14 +28,25 @@ acknowledge or cite QGreenland and the National Snow and Ice Data Center in your
 
 Suggested citations for QGreenland:
 
-Version 1.0:
-Moon, T., Fisher, M., Harden, L., and T. Stafford (2021). QGreenland (v1.0.0) [software],
-[National Snow and Ice Data Center](https://qgreenland.org/).
 
-Version 2.0: 
-Moon, T., Fisher, M., Harden, L., Simonoko, H., and T. Stafford (2022). QGreenland (v2.0.0) [software],
-[National Snow and Ice Data Center](https://qgreenland.org/).
+### Version 1.0
+
+> Moon, T., Fisher, M., Harden, L., and T. Stafford (2021). QGreenland (v1.0.0) [software],
+> [National Snow and Ice Data Center](https://qgreenland.org/).
+
+
+### Version 2.0
+
+> Moon, T., Fisher, M., Harden, L., Simonoko, H., and T. Stafford (2022). QGreenland (v2.0.0) [software],
+> [National Snow and Ice Data Center](https://qgreenland.org/).
+
+
+### Version 3.0
+
+> Moon, T., Fisher, M., Thurber, A., and T. Stafford (2023). QGreenland (v3.0.0) [software],
```

Comment:
This citation style strikes me as weird. The last name is different than the rest of the names.

Also, should we be listing ourselves as primary authors in the `[software]` citation?

Reply:
Hmm, not sure why we did it this way. Probably best to double-check with Twila on citation format and author order?

Reply:
@twilamoon-science do you think we have the citation format correct? I followed the pattern from our v1 and v2 citations, but the pattern is a bit confusing to me :) I think it was alphabetical (aside from PI) before, so I should have put Stafford before Thurber. It's weird to to reverse the initial style only for the last entry in the list... or is it after the first three? It's not like reversing the initial saves any space...

Reply:
Also, please see the first post in this thread RE: `[software]` citation. Should Trey and I be listed as primary authors?

Reply:
#702 to continue discussion. Let's merge this :)